rails_app/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+Uq1NfPnEhpn8FbIQdm14D7BzkHJ64VcpARsdjBAXFmT0Ccj+PMEZpsG51uQgqlLYR4uZ1s2uDfj2bPSzI7eT+6twJK17XP/L0K8Pe2wnJWBOvlhJlycYS3G5mmwMKx3/G33z/ppj8hnO+YS0GN2a5gAB4jq42RG0BlsaTQsSIIh/THKWWSrfkpoU94+TLCitL/tWB7fXMnbJRpOf71k/ouyYdnwDItB4o3MB10BNJR+x+lkmRPLELs/uWeuoEnGD+QitkOnehtpc6zgEqdDrpo578KEBjJqUnNs10BJ5+wK8XA4Y8yRU71OHbD1zrzsTHnvQIkUXNyZPSzRp+ozC9wJiIEQ5IaCfkjFL8eOGQAhF3Jc91CjHOY8cijG8SJvv5KmbBwUSPwdohXRMFLyE13ALQ1KVb9dQjnk3hyPYN8oZm8Djwx6IiOWwfDwTt/uKq3N9AQw9i/dKFCG67u7tN+DwxgwSiabkiyfAZNW8XMm78VR+d3V12lL5--tif5D9lF4qliyT4K--OO/Tit2HXIll6afNpTdwZg==
```

Comment:
Is it intentional to commit this file _for a sample_? If it is, shouldn't we also document the corresponding encryption key somewhere?